### PR TITLE
Added Longest Common Subsequence [JavaScript]

### LINKED ISSUE #388: Longest Common Subsequence [JavaScript]
#### This issue is about [Longest Common Subsequence](http://www.geeksforgeeks.org/dynamic-programming-set-4-longest-common-subsequence).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
Fixes #388 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by Anshuman-Verma** [comment]
**DO NOT REVIEW**

**Comment by aashutoshrathi** [comment]
@Anshuman-Verma Resolve conflicts

**Comment by Anshuman-Verma** [comment]
Done @aashutoshrathi

**Comment by Anshuman-Verma** [comment]
@aa

File: longest_common_subsequence/longestCommonSubsequence.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+function longestCommonSubsequence (str1, str2) {
+	/*
+	* Time Complexity - O( len(str1) * len(str2) )
+	* Space Complexity - O( len(str1)*len(str2) )
```
Comment (Monal5031) [inline longest_common_subsequence/longestCommonSubsequence.js]: No partiality here please, add space around binary operator as added above 😛

Comment (Anshuman-Verma) [inline longest_common_subsequence/longestCommonSubsequence.js]: Damn, partiality :stuck_out_tongue: 
Fixed

**Pull request merged**